src/foam/nanos/notification/NotificationListView.js

```diff
@@ -0,0 +1,157 @@
+foam.CLASS({
+  package: 'foam.nanos.notification',
+  name: 'NotificationListView',
+  extends: 'foam.u2.View',
+  documentation: 'Notification controller',
+
+  implements: [
+    'foam.mlang.Expressions'
+  ],
+
+  imports: [
+    'notificationDAO',
+    'stack',
+    'user'
+    ],
+
+  requires: [
+    'foam.nanos.notification.Notification',
+    'foam.u2.stack.Stack',
+    'foam.nanos.notification.NotificationRowView'
+  ],
+
+  exports: [
+    'as data',
+    'notificationDAO'
+  ],
+
+  css: `
+  
+    .foam-u2-DAOList > div {
```

Comment:
You haven't scoped this CSS so it will affect all DAOLists globally. Use ^ to scope.